src/caa-cases/containers/caa-cases/caa-cases.component.html

```diff
@@ -48,22 +48,19 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
       <br>
       Your colleague must be registered with your organisation and have access to Manage Cases.
     </p>
-    <div *ngIf="hasResults()" class="pagination-top"
-      attr.aria-label="{{ getTotalResults() }} results have been found" role="status">
-      <span class="text-16" id="summary__text">Showing
-        <span class="govuk-!-font-weight-bold">{{ getFirstResult() }}</span>
-        to
-        <span class="govuk-!-font-weight-bold">{{ getLastResult() }}</span>
-        of
-        <span class="govuk-!-font-weight-bold">{{ getTotalResults() }}</span> results</span>
-    </div>
     <mat-tab-group animationDuration="0ms" (selectedTabChange)="tabChanged($event)" *ngIf="navItems && navItems.length > 0">
       <mat-tab *ngFor="let navItem of navItems; let last = last;" [label]="navItem.text">
-        <button *ngIf="navItems && navItems.length > 0 && (cases$ | async)?.length > 0"
-          [disabled]="(shareCases$ | async)?.length === 0" id="btn-share-button" class="govuk-button"
-          data-module="govuk-button" type="button" aria-haspopup="true" aria-expanded="false" (click)="shareCaseSubmit()">
-          Share case
-        </button>
+        <div class="pagination-info" *ngIf="hasResults()">
+          <div>
+            <p class="govuk-body">Showing <strong>{{ getFirstResult() }}</strong> to <strong>{{ getLastResult() }}</strong> of <strong>{{ getTotalResults() }} {{ currentCaseType }} cases</strong></p>
```

Comment:
@johnbenjamin-hmcts - is it intentional to have the word "cases" in *bold* type at the end? For example, "Showing **1** to **5** of **10 Divorce cases**"

Reply:
@Daniel-Lam I feel the same, but followed the prototype and the screenshots. Shall we go with this for now and I will check with Tommy and update in later tickets if required.